proposals/lip-0044.md

```diff
@@ -54,15 +54,16 @@ In this section, we specify the substores that are part of the Validators module
 
 We define the following constants:
 
-Name                           | Type    | Value                             | Description
------------------------------- | ------- | --------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-`MODULE_ID_VALIDATORS`         | uint32  | TBD                               | ID of the Validators module.
-`STORE_PREFIX_VALIDATORS_DATA` | bytes   | 0x0000                            | Store prefix of the validators data substore.
-`STORE_PREFIX_GENERATOR_LIST`  | bytes   | 0x4000                            | Store prefix of the generator list substore.
-`STORE_PREFIX_BLS_KEYS`        | bytes   | 0x8000                            | Store prefix of the registered BLS keys substore.
-`STORE_PREFIX_GENESIS_DATA`    | bytes   | 0xc000                            | Store prefix of the genesis data substore.
-`INVALID_BLS_KEY`              | bytes   | 48 bytes all set to 0x00          | An invalid BLS key, used as a placeholder before a valid BLS key is registered. It is invalid as deserialization fails since the most significant bit of the first byte is zero while the total length is 48 (see second point [here](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-pairing-friendly-curves-07#appendix-C.2)).
-`BLOCK_TIME`                   | integer | 10 (value for the Lisk mainchain) | Block time (in seconds) set in the chain configuration.
+Name                           | Type    | Value                                                | Description
+------------------------------ | ------- | ---------------------------------------------------- | -------------------------------------------------------------------------------
+`MODULE_ID_VALIDATORS`         | uint32  | TBD                                                  | ID of the Validators module.
+`STORE_PREFIX_VALIDATORS_DATA` | bytes   | 0x0000                                               | Store prefix of the validators data substore.
+`STORE_PREFIX_GENERATOR_LIST`  | bytes   | 0x4000                                               | Store prefix of the generator list substore.
+`STORE_PREFIX_BLS_KEYS`        | bytes   | 0x8000                                               | Store prefix of the registered BLS keys substore.
+`STORE_PREFIX_GENESIS_DATA`    | bytes   | 0xc000                                               | Store prefix of the genesis data substore.
+`INVALID_BLS_KEY`              | bytes   | 48 bytes all set to 0x00                             | An invalid BLS key, used as a placeholder before a valid BLS key is registered.
```

Comment:
I would expect something `INVALID` to cause error. Maybe we can say `UNREGISTERED_BLS_KEY` as in the explanation? Or `UNSET_BLS_KEY`.

Reply:
We use consistently the term `INVALID` across several LIPs: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md and also in the new migration LIP, therefore I would not change it now.